lib/metallb/kustomization.yaml

```diff
@@ -3,4 +3,3 @@ kind: Component
 
 resources:
 - metallb_deploy.yaml
```

Comment:
Not clear if there is real need(atleast not for OSPRH-3743) to split the deployment of metallb and nmstate, could have lived together and have 1 less step.

Reply:
In an earlier version of architecture metalb and nmstate were separated and checked separately. I don't think it hurts to have them separate. 

If we want to make it one step I'd edit `architecture/examples/common` to have one directory `nmstate_metallb` with both components (lib/metallb + lib/nmstate). I didn't like the overly generic `deploy` name. Let's see what other reviewers think. I'll focus on splitting out NNCP for now.